src/content/en/updates/2017/06/credential-management-updates.md

```diff
@@ -0,0 +1,333 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Latest Updates to the Credential Management API
+
+{# wf_updated_on: 2017-06-07 #}
+{# wf_published_on: 2017-06-07 #}
+{# wf_tags: performance #}
+{# wf_featured_snippet: Latest updates coming to the Credential Management API in Chrome 60. Also includes an update landed in Chrome 57. #}
+
+# Latest Updates to the Credential Management API {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+
+Some of the updates described here are explained at the Google I/O session,
+**Secure and Seamless Sign-In: Keeping Users Engaged**:
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="DBBFK7bvEQo"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+## Chrome 57
+
+Chrome 57 introduced this important change to the
+[Credential Management API](/web/fundamentals/security/credential-management).
+
+### Credentials can be shared from a different subdomain 
+
+Chrome can now retrieve a credential stored in a different subdomain using the
+[Credential Management API](/web/fundamentals/security/credential-management).  
+
+For example, if a password is stored in `login.example.com`,
+a script on `www.example.com` can show it as one of account items in account chooser dialog.
+When a user taps on the dialog, the password gets passed.
+
+Explicitly store the password using `navigator.credentials.store()`.
+Once it's stored, the password is available as a credential
+in the exact same origin `www.example.com` onward.
+
+**You must explicitly store the credential to the origin.**
+Users choosing the credentials won’t copy them to the current origin.
+
+In the following screenshot, credential information stored under `login.aliexpress.com`
+is visible to `m.aliexpress.com` and available for the user to choose from:
+
+<figure>
+  <img src="/web/updates/images/2017/06/credentials.png"
+       alt="Account chooser showing selected subdomain login details"/>
+</figure>
+
+<aside>
+  <strong>Coming soon:</strong>
+  Sharing credentials between totally different domains is in development.
+</aside>
+
+## Chrome 60
+
+Chrome 60 introduces several important changes to the
+[Credential Management API](/web/fundamentals/security/credential-management):
+
+* `PasswordCredential` object now includes a password.
```

Comment:
Any idea how to do this? Do I have to add a table of contents in order to be able to access the headings? I tried a simple test of linking to what I thought the link might be for one section, and that didn't seem to work. General comment-- table of contents don't seem to show up in testing site, so it's a little tricky to see what the headings might be. Help much appreciated on best way to approach this.